resource/abstractions/individual_access.md

```diff
@@ -0,0 +1,96 @@
+---
+layout: default
+grand_parent: General Resource Admin
+parent: Abstractions
+---
+
+# Individual Access
+
+General Resource Profile Administration functions for checking an individual user's access to a General Resource Profile. 
+{: .fs-6 .fw-300 }
+
+## `ResourceAdmin.get_my_access()`
+
+```python
+def get_my_access(self, resource: str, class_name: str) -> Union[str, bytes, None]:
+```
+
+#### 📄 Description
+
+Check **Your Access** for a general resource profile.
+
+#### 📥 Parameters
+* `resource`<br>
+  The **general resource profile** for which RACF should check the current user's access level.
```

Comment:
Remove `for`.

Reply:
Maybe something like this?

```markdown
The **general resource profile** to get the current user's access level for.
```

Reply:
I still don't love that, but I will change it. I do think that a comprehensive review of the doc to make these into complete sentences will avoid a lot of my "wording" complaints and ill-feelings about these parameter descriptions.